src/distributions/weighted.rs

```diff
@@ -130,6 +132,342 @@ impl<X> Distribution<usize> for WeightedIndex<X> where
     }
 }
 
+/// A distribution using weighted sampling to pick a discretely selected item.
+///
+/// Sampling an [`AliasMethodWeightedIndex<W>`] distribution returns the index
+/// of a randomly selected element from the vector used to create the
+/// [`AliasMethodWeightedIndex<W>`]. The chance of a given element being picked
+/// is proportional to the value of the element. The weights can have any  type
```

Comment:
There is a redundant whitespace in front of "type".

Reply:
Fixed in 9c44b6aa510b25408ce050d2785b5a9fcb4e98de.